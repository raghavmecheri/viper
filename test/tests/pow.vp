float a = pow2(2.0);
print(a);

float b = pow2(a);

print(b);

float c = pow2(876.0);
print(c);


