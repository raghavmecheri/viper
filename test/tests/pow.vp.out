4
16
767376